testing/fixtures/fixures.go

```diff
@@ -2,34 +2,35 @@ package fixtures
 
 import "testing"
 
-type Setup func(dir string) error
-type Assertion func(dir string, actualOutput string, t *testing.T) bool
-type Teardown func(dir string) error
+type setup func(dir string) error
+type assertion func(dir string, actualOutput string, t *testing.T) bool
+type teardown func(dir string) error
 
-type Test struct {
+type test struct {
```

Comment:
@RajatVaryani thanks for the PR.


struct `test` is a definition used across test package, how about making it public `Test` - with capital T?

WDYT?

Reply:
Yes Made the changes.